lms/djangoapps/teams/static/teams/js/spec/views/topics_spec.js

```diff
@@ -57,5 +59,23 @@ define([
             expect(footerEl.text()).toMatch('1\\s+out of\\s+\/\\s+2');
             expect(footerEl).not.toHaveClass('hidden');
         });
+
+        it('refreshes the topics when a team is created', function() {
+            var requests = AjaxHelpers.requests(this),
+                topicsView = createTopicsView();
+
+            topicsView.collection.teamEvents.trigger('teams:update', { action: 'create' });
+            topicsView.render();
+            AjaxHelpers.expectJsonRequestURL(
+                requests,
+                'api/teams/topics',
+                {
+                    course_id : 'my/course/id',
+                    page : '1',
+                    page_size : '5',  // currently the page size is determined by the size of the collection
+                    order_by : 'name'
+                }
+            );
+        });
```

Comment:
It would be good to do some simple verification that the view refreshed correctly (I know my test didn't do this). As it stands, it might be firing off an AJAX call but dropping the results on the floor.